README.md

```diff
@@ -8,7 +8,7 @@ $ mdcat sample.md
 
 ![mdcat showcase with different colour themes][sxs]
 
-mdcat in [WezTerm], with "One Light (base16)", "Gruvbox Light", and "Darcula
+mdcat in [WezTerm], with "One Light (base16)", "Gruvbox Light", and "Dracula
```

Comment:
This was intentional, see https://wezfurlong.org/wezterm/colorschemes/d/index.html?h=darcula#darcula-base16